index.html

```diff
@@ -76,796 +64,17 @@ <h1>Tech Radar</h1>
-        "link": "https://learn.microsoft.com/en-us/azure/cosmos-db/",
-        "status": 0
-      },
-      {
-        "quadrant": 0,
-        "ring": 1, //probeer
-        "label": "PostgreSQL",
-        "link": "https://en.wikipedia.org/wiki/PostgreSQL",
-        "status": 0
-      },
-      {
-        "quadrant": 0,
-        "ring": 2, //onderzoek
-        "label": "Oracle Cloud",
-        "link": "https://www.oracle.com/cloud/",
-        "status": 0
-      },
-      {
-        "quadrant": 0,
-        "ring": 2, //onderzoek
-        "label": "Github Server",
-        "link": "https://docs.github.com/en/enterprise-server@3.5/admin/overview/about-github-enterprise-server",
-        "status": 0
-      },
-      {
-        "quadrant": 0,
-        "ring": 2, //onderzoek
-        "label": "Redis",
-        "link": "https://en.wikipedia.org/wiki/Redis",
-        "status": 0
-      },
-      {
-        "quadrant": 0,
-        "ring": 2, //onderzoek
-        "label": "Dev Container",
-        "link": "https://code.visualstudio.com/docs/devcontainers/containers",
-        "status": 0
-      },
-      {
-        "quadrant": 2,
-        "ring": 1, //probeer
-        "label": "Green Computing",
-        "link": "https://en.wikipedia.org/wiki/Green_computing",
-        "status": 0
-      },
-      {
-        "quadrant": 0,
-        "ring": 3, //verminder
-        "label": "DFS",
-        "link": "https://en.wikipedia.org/wiki/Distributed_File_System_(Microsoft)",
-        "status": 0
-      },
-      {
-        "quadrant": 0,
-        "ring": 3, //verminder
-        "label": "UC4",
-        "link": "https://www.broadcom.com/products/software/automation/automic-automation",
-        "status": 0
-      },
-          {
-        "quadrant": 0,
-        "ring": 3, //verminder
-        "label": "AKS",
-        "link": "https://learn.microsoft.com/en-us/azure/aks/intro-kubernetes",
-        "status": 2
-      },
-      {
-        "quadrant": 0,
-        "ring": 3, //verminder
-        "label": "AVS*",
-        "link": "avs.html",
-        "status": 0
-      },
-      {
-        "quadrant": 0,
-        "ring": 3, //verminder
-        "label": "BizTalk",
-        "link": "https://en.wikipedia.org/wiki/Microsoft_BizTalk_Server",
-        "status": 0
-      },
-      {
-        "quadrant": 0,
-        "ring": 3, //verminder
-        "label": "MSMQ",
-        "link": "https://learn.microsoft.com/en-us/previous-versions/windows/desktop/msmq/ms703216(v=vs.85)",
-        "status": 0
-      },
-      {
-        "quadrant": 0,
-        "ring": 3, //verminder
-        "label": "Active Directory",
-        "link": "https://en.wikipedia.org/wiki/Active_Directory",
-        "status": 0
-      },
-      {
-        "quadrant": 0,
-        "ring": 3, //verminder
-        "label": "AVD",
-        "link": "https://learn.microsoft.com/en-us/azure/virtual-desktop/overview",
-        "status": 0
-      }
-    ]
-});
+  const currentVersion = "202307";
```

Comment:
De volgende verbetering zou zijn om een extra json toe te voegen met alle beschikbare versies en die als eerste in te laden. Daaruit kun je dan ook afleiden wat de laatste is (sorteren op datum).

Reply:
Done :)